chapters/07-build-management.Rmd

```diff
@@ -6,10 +6,31 @@
 - Pre-push running test, build, and check
 - Continuous integration using GitHub Actions
 
-
+<!-- NOTE: Preferably something linked to working towards completing the project assignments, maybe to be used as a milestone? -->
 
 ## Final exercise
 
-<!-- NOTE: Preferably something linked to working towards completing the project assignments, maybe to be used as a milestone? -->
+The purpose of this exercise is for the participant to go through a cycle of iteration of:
+
+- 1) checking if their package succeeds with R CMD
+- 2) adding a function and the related steps
+- 3) check if the package succeeds R CMD with the new additions
+- 4) deploy a GitHub Action that performs the R CMD check every time they push to GitHub. 
 
+The individual steps look like this:
 
+1. Check that the package builds with the current functions
+  - `devtools::check()`
+2. Add a function called `FIXME`
+  - `usethis::use_r("FIXME")`
```

Comment:
What do you think about having `devtools::check()` as part of each of these steps?  E.g. you can pick up dependency issues at this step, before having to worry about documentation issues.